addon/-private/execution_context/helpers.js

```diff
@@ -36,3 +36,37 @@ export function fillElement(selection, content, { selector, pageObjectNode, page
     $selection.val(content);
   }
 }
+
+/**
+ * @private
+ *
+ * Given an element, asserts that element is focusable/blurable
+ *
+ * @param {jQuery} element - jQuery object containing the element(s) to check
```

Comment:
I don't feel super strongly, but I generally prefer calling this `$selection` or `$collection` or something. I tend to think when I see a variable or param called `element` that someone is calling it out as a native DOM element as opposed to a jQuery object, and that's generally the case in this codebase.